10/umbraco-deploy/troubleshooting.md

```diff
@@ -28,6 +28,18 @@ In some cases, a hard limit imposed by the Cloud hosting platforms such as Azure
 
 If you find the process slow or reporting a platform timeout, there are a few options you can take.
 
+### Consider partial restore
+
+If restoring, you can choose to pull down a smaller set of content via the partial restore feature. With this you select an item in the remote environment to restore. You can select to include child items. Any items related to the selected ones, for example via content or media pickers, will also be restored.
+
+### Consider import/export
+
+In addition to transfers via the backoffice, content (and schema) can be moved between environments via Deploy's import/export feature. With this a selection of Umbraco data can be exported from one environment to a `.zip` file. That file can then be imported into another environment.
```

Comment:
```suggestion
In addition to transfers via the backoffice, content, and schema can be moved between environments via Deploy's import/export feature. With this, a selection of Umbraco data can be exported from one environment to a `.zip` file. That file can then be imported into another environment.
```

Reply:
Not sure the suggestion is quite right actually.  Here what I'm trying to say is... "you normally transfer content via the backoffice, but you can also use the import/export feature, and this allows for schema too".  If you can find away of rephrasing to get that meaning across, that would be best I think.

Reply:
Ahhh, thanks for elaborating!

What do you think about something like this:

_In addition to transferring content via the backoffice, it is possible to move both content and schema between environments via Deploy's import/export feature. With this, a selection of Umbraco data can be exported from one environment to a `.zip` file. That file can then be imported into another environment._

Reply:
Sounds perfect.  Will you update or would you prefer I do?

Reply:
Great! I just updated my suggestions with the change, so you should be able to just commit the suggestions :D